subsystem/low_level_api_test.go

```diff
@@ -14,7 +14,7 @@ import (
 )
 
 var _ = Describe("Low level API behaviours", func() {
-	It("Rejects unknown JSON fields", func() {
+	It("low_level_api Rejects unknown JSON fields", func() {
```

Comment:
adding a flag in order to run a focused subsystem-test

Reply:
Can it be added to the `Describe` block? Or will this be removed before merge?

Reply:
done